doc/source/examples/cfproto_mnist.ipynb

```diff
@@ -24,6 +24,20 @@
     "Other options include generating counterfactuals for specific classes or including trust score constraints to ensure that the counterfactual is close enough to the newly predicted class compared to the original class. Different use cases are illustrated throughout this notebook."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "To enable support for CounterfactualProto, you may need to run\n",
+    "    \n",
+    "```bash\n",
+    "pip install alibi[tensorflow]\n",
+    "```\n",
+    "\n",
```

Comment:
Same comment as above, can we make the box title be "Note" and the text part of the body?